src/components/view-item/index.js

```diff
@@ -54,9 +56,15 @@ const ViewItem = ({
             break;
 
         case 'potion':
+            data.hp = data.hpReset;
+            let potionRestore = calculateWisdomPotionBonus(
+                data.hp,
+                calculateModifier(stats.abilities.wisdom)
+            );
+            data.hp = potionRestore;
             itemStats.push(
                 <StatsItem
-                    stats={{ name: 'heal', value: data.hp }}
```

Comment:
On a second look through, "data.hp" didn't have to be changed to "potionRestore" as they would have done the same thing. Can change back if required.

Reply:
Would that also remove the need for the hpReset?

Reply:
So the purpose of hpReset was that closing and reopening of a potion object continuously recalls the function, so the amount to be restored continues to increase when it should only be static and run once. Therefore, I implemented the hpReset attribute to reset to its initial value. It was also simpler to alter the data.hp value for the amount to be restored.

Reply:
Yeah, but, if you remove the 'data.hp = hpRestore' line, then wouldn't that keep the value consistent? Or am I misunderstanding?

Reply:
When using a potion, the health restored is taken from data.hp, so the wisdom bonus will need to adjust the value of data.hp to account for the new value.

Reply:
Ohh, okay. Then yeah, it's up to you if you want to change what you said. I don't think it makes much difference.

Reply:
Ohh, okay. Then yeah, it's up to you if you want to change what you said. I don't think it makes much difference.

Reply:
It doesn't need to be changed imo as it does the same thing but if it irritates some people then I can change it.